sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/HashAggregateExec.scala

```diff
@@ -54,8 +54,7 @@ case class HashAggregateExec(
     resultExpressions: Seq[NamedExpression],
     child: SparkPlan)
   extends BaseAggregateExec
-    with BlockingOperatorWithCodegen
-    with AliasAwareOutputPartitioning {
+    with BlockingOperatorWithCodegen {
```

Comment:
nit: could you correct the indentation in the style guide? https://github.com/databricks/scala-style-guide#spacing-and-indentation

Reply:
@maropu - sure, updated.